opflexagent/gbp_ovs_agent.py

```diff
@@ -577,6 +580,35 @@ def main():
     common_config.setup_logging()
     q_utils.log_opt_values(LOG)
 
+    agent_mode = cfg.CONF.OPFLEX.agent_mode
+    if agent_mode == 'dvs':
+        main_dvs()
+    else:
+        main_opflex()
+
+
+def main_dvs():
```

Comment:
main_dvs() and main_opflex() share quite a few common codes....Can we just call into the same main function but with a flag to tell the difference?